libs/adapters/src/express/command.ts

```diff
@@ -21,10 +21,15 @@ export const expressCommand =
       }
     >,
     res: Response<Partial<T> | undefined>,
+    next: NextFunction,
   ) => {
-    const results = await command(md, req.params.id, req.body, {
-      user: req.user as User,
-      address_id: req.body.address_id,
-    });
-    return res.json(results);
+    try {
```

Comment:
We have configured express to handle async errors (see main.ts)
```
// set up express async error handling hack
require('express-async-errors');
```

Is there any reason why we need to explicitly try catch here? (Same argument with query handler)

Reply:
I didn't want to pollute the existing handlers with new error types and logic... this is completely detached but using a similar pattern. Once we start implementing and replacing routes, we'll merge both.

Reply:
Makes sense to me.